jobs/web/spec

```diff
@@ -1249,6 +1249,10 @@ properties:
     env: CONCOURSE_CONJUR_CERT_FILE
     description: |
       Path to cert file used if conjur instance is using a self-signed cert.
+  conjur.ssl_certificate:
+    env: CONCOURSE_CONJUR_SSL_CERTIFICATE
```

Comment:
Instead of adding this, I think we should remove `conjur.cert_file`, since it's not really useful at the moment, and add a `conjur.tls.ca_cert` property which would match all the other cert configs.

Something like this:

```yaml
conjur.tls.ca_cert:
  type: certificate
  env_fields: {certificate: {env_file: CONCOURSE_CONJUR_CERT_FILE}}
  description: # ...
```

This would match how we configure CA certs elsewhere, e.g. vault:

https://github.com/concourse/concourse-bosh-release/blob/ec4aa674aab8dc3155651df5276e70f7dd61d078/jobs/web/spec#L1192-L1196

Note that this is `type: certificate`, so in order to use this you'd specify a `certificate` field in your manifest like so:

```yaml
properties:
  conjur:
    tls:
      ca_cert:
        certificate: |
          -- BEGIN BLAH BLAH BLAH --
```

(It's a little more verbose but it lets people use typed, generated values with `variables:`.)

Reply:
What does this mean? `env_fields: {certificate: {env_file: CONCOURSE_CONJUR_CERT_FILE}}` I am just concerned right now because concourse expects to read the certificate from a file. The file path is defined within the environment variable CONCOURSE_CONJUR_CERT_FILE. I was thinking about adding another environment variable that called something like CONCOURSE_CONJUR_SSL_CERTIFICATE which would be the actual content of the certificate as a string.
Does that make sense?

Reply:
I guess what I don't understand is for Vault, how does `CONCOURSE_VAULT_CA_CERT` map to https://github.com/concourse/concourse/blob/master/atc/creds/vault/manager.go#L35.

Shouldn't the environment variable be `CONCOURSE_VAULT_TLS_CA_CERT`?

Or is the TLS ignored because this line ends with `squash`: https://github.com/concourse/concourse/blob/master/atc/creds/vault/manager.go#L26

Reply:
I resolved this in the most recent commit. Let me know if you need anything else.